tests/end_to_end/andor_assignment/irix-o2-out.c

```diff
@@ -62,9 +62,7 @@ s32 test(s32 arg0, s32 arg1, s32 arg2, s32 arg3) {
             phi_v1_3 = phi_v1_6 + 5;
         }
     }
-    phi_v1_4 = phi_v1_3;
-    phi_v1_7 = phi_v1_3;
-    if ((phi_s0_2 != 0) && (phi_t0_2 != 0) && (sp24 = phi_v1_3, (func_00400090(phi_s0_2 + phi_t0_2) != 0)) && (arg3 != 0)) {
+    if ((phi_s0_2 != 0) && (phi_t0_2 != 0) && (sp24 = phi_v1_3, phi_v1_4 = phi_v1_3, phi_v1_7 = phi_v1_3, (func_00400090(phi_s0_2 + phi_t0_2) != 0)) && (arg3 != 0)) {
```

Comment:
do you know why this change happened? I don't understand it

Reply:
The function call `func_00400090` saves/restores the register `v1`. [When the register is restored, it is wrapped in an `EvalOnceExpr`](https://github.com/matt-kempster/mips_to_c/blob/master/src/translate.py#L3421-L3435) which is  no longer unwrapped.